website/docs/r/redshiftserverless_namespace.html.markdown

```diff
@@ -24,7 +24,10 @@ This resource supports the following arguments:
 
 * `admin_password_secret_kms_key_id` - (Optional) ID of the KMS key used to encrypt the namespace's admin credentials secret.
 * `admin_user_password` - (Optional) The password of the administrator for the first database created in the namespace.
-  Conflicts with `manage_admin_password`.
+  Conflicts with `manage_admin_password` and `admin_user_password_wo`.
+* `admin_user_password_wo` - (Optional, Write-Only) The password of the administrator for the first database created in the namespace.
```

Comment:
The `manage_admin_password` argument needs to now include a conflict with `admin_user_password_wo`.